src/prof_data.c

```diff
@@ -1162,9 +1303,18 @@ prof_tctx_destroy(tsd_t *tsd, prof_tctx_t *tctx) {
 
 	assert(tctx->cnts.curobjs == 0);
 	assert(tctx->cnts.curbytes == 0);
+	/*
+	 * These asserts are not correct -- see the comment about races in
+	 * prof.c
+	 *
+	 * assert(tctx->cnts.curobjs_shifted_unbiased == 0);
+	 * assert(tctx->cnts.curbytes_unbiased == 0);
+	 */
```

Comment:
Same here.

Reply:
These are just to give context for the preceding comment -- letting the future reader know that they weren't excluded because of an oversight.

Reply:
Same here.

Reply:
Same here.